### PR TITLE
keepAliveTimeout: introduce `System default`

### PR DESCRIPTION
**Pull request by mwelzl**
This partially addresses #1428
Please see the discussion there for the rationale behind this.

### CODE REVIEW

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -2160,10 +2160,10 @@ Name:
 : keepAliveTimeout
 
 Type:
-: Numeric (non-negative) or `Disabled`
+: Numeric (non-negative) or `Disabled` or `System default`
```
Comment (philsbln) [inline draft-ietf-taps-interface.md:2163]: I guess having a single-word value makes this more portable…

```suggestion
: Numeric (non-negative) or `Disabled` or `Default`
```

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -2160,10 +2160,10 @@ Name:
...
 
 Default:
-: `Disabled`
+: `System default`
```
Comment (philsbln) [inline draft-ietf-taps-interface.md:2166]:
```suggestion
: `Default`
```

File: draft-ietf-taps-interface.md [hunk 3]
```diff
@@ -2173,7 +2173,9 @@ will only take effect when the underlying stack supports sending keep-alive pack
 Guidance on setting this value for connection-less transports is
 provided in {{?RFC8085}}.
 A value greater than the Connection timeout ({{conn-timeout}}) or the enumerated value `Disabled` will disable the sending of keep-alive packets. A Transport Services API
-could express `Disabled` in an environment-typical way, e.g., as a Union type or special value.
+could express `Disabled` and `System default` in an environment-typical way, e.g., as a Union type or special value. Choosing `System default` is a hint to the Transport Services system that it should
```
Comment (philsbln) [inline draft-ietf-taps-interface.md:2176]:
```suggestion
could express `Disabled` and `Default` in an environment-typical way, e.g., as a Union type or special value. Choosing `Default` is a hint to the Transport Services system that it should
```

**Review by philsbln — APPROVED** [review]
Works for me except that I really prefer a single-word here

**Comment by mwelzl** [comment]
How about SysDefault instead of Default? It's a single word, yet avoids the somewhat ugly:
"Default: default"

File: draft-ietf-taps-interface.md [hunk 4]
```diff
@@ -2160,10 +2160,10 @@ Name:
 : keepAliveTimeout
 
 Type:
-: Numeric (non-negative) or `Disabled`
+: Numeric (non-negative) or `Disabled` or `SysDefault`
 
 Default:
-: `Disabled`
+: `SysDefault`
```
Comment (tfpauly) [inline draft-ietf-taps-interface.md:2166]: The default for keepalives should be off.

**Comment by tfpauly** [comment]
I think we should close as not to be fixed

**Review by tfpauly — CHANGES_REQUESTED** [review]
Let's discuss and maybe close

**Review by philsbln — CHANGES_REQUESTED** [review]
After discussion, I second @tfpauly

**Pull request closed without merging**